### DISCUSSION TITLE
download

### DISCUSSION THREAD

**Discussion by KapilBagdi** [Laravel-Backpack/community-forum#420] [General] [answered]
im able to download pdf but its showing reloading again and again
CRUD::column('title');

        // but since you've probably already defined columns in your List or Show operation, you could do:
        $this->setupListOperation();

        // in addition, in case you want to change settings:
        CRUD::set('download.view', 'user.invoice.wrapper'); // default is 'crud::show'
        CRUD::set('download.format', 'A4'); // default is 'A4'
        CRUD::set('download.headers', ['Content-Type' => 'application/pdf']); // default is 'A4'

this was my above code
        
        cutrrently im using employee_companies datatable so allthe name of my file and folder is accordingly

**Comment by tabacitu** [comment, marked as answer]
Hmm.... @KapilBagdi you are not supposed to use the example code as-is. For example, the fifth line tells the operation to use the `user.invoice.wrapper` blade view - I don't think that exists in your project, it was just an example. It should probably work if you comment out those 3 last lines.

If you still get an endless refresh, take a look at the log files - when Browsershot has a problem, it logs what happened, so it'll be easier to tell what _exactly_ went wrong from there.

Hope it helps!
